src/website/app/demos/Text/examples/nested.js

```diff
@@ -0,0 +1,73 @@
+/* @flow */
+import palette from '../../../../../library/colors';
+import { createStyledComponent } from '../../../../../library/styles';
+import { mineralTheme as theme } from '../../../../../library/themes';
+import _Text from '../../../../../library/Text';
+
+const DemoLayout = createStyledComponent(
+  'div',
+  {
+    '& > *': {
+      marginBottom: '1em'
+    }
+  },
+  {
+    includeStyleReset: true
+  }
+);
+
+const Text = createStyledComponent(_Text, ({ theme }) => ({
+  backgroundColor: palette.indigo_20,
+  padding: theme.space_inset_sm
+})).withProps({ noMargins: true });
+
+export default {
+  id: 'nested',
+  title: 'Nested Components',
+  description: `Nested Text components will display with the expected styles and
+semantic elements by default. You can control this behavior with the \`inherit\`
+prop.
+
+<Callout title="Note">
+  <p key={0}>
+    For illustrative purposes, a purple background color and padding have been
+    applied to the Text components below.
+  </p>
+</Callout>`,
+  scope: { createStyledComponent, DemoLayout, palette, Text, theme },
```

Comment:
Again, suggest to rename `palette` to `colors` to match the folder name and perhaps make a better connection for the user.